openwpm/config.py

```diff
@@ -0,0 +1,263 @@
+import os
+from dataclasses import dataclass, field
+from typing import List, Tuple
+
+from dataclasses_json import dataclass_json
+
+from .errors import ConfigError
+
+BOOL_TYPE_VALIDATION_LIST = [True, False]
+DISPLAY_MODE_VALIDATION_LIST = ["native", "headless", "xvfb"]
+SUPPORTED_BROWSER_LIST = [
+    "firefox"
+]  # Using List instead of a str type to future proof the logic as OpenWPM may add support for more browsers in future
```

Comment:
Fair even though this is unlikely to happen.
I'll likely file a follow up issue to remove the `browser` in `browser_params`

Reply:
> Fair even though this is unlikely to happen.

Is this because of compatibility issues?

Reply:
Yes, our WebExtension, that we use to collect all of our data, relies heavily on Firefox internal APIs, so we can't use it for other browsers.